### PR TITLE
systemd-firstboot: add page

### PR DESCRIPTION
**Pull request by varadrane7**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/systemd-firstboot.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# systemd-firstboot
+
+> Initialize basic system settings on or before the first boot-up of a system.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-firstboot.html>.
+
+- Set the root directory for the basic system settings initialization:
+
+`sudo systemd-firstboot --root={{root_directory_path}}`
```
Comment (kbdharun) [inline pages/linux/systemd-firstboot.md:8]:
```suggestion
`sudo systemd-firstboot --root={{path/to/root_directory}}`
```

File: pages/linux/systemd-firstboot.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Prompt the user interactively for a specific basic setting:
+
+`sudo systemd-firstboot --prompt={{setting}}`
+
+- Force writing configuration even if the relevant files already exist:
+
+`sudo systemd-firstboot --force`
+
+- Remove all existing files that are configured by systemd-firstboot:
```
Comment (kbdharun) [inline pages/linux/systemd-firstboot.md:30]:
```suggestion
- Remove all existing files that are configured by `systemd-firstboot`:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution and welcome to `tldr`. I have some suggestions for this page.

**Review by einverne — APPROVED** [review]
Looks good to me after @kbdharun 's suggestions.

File: pages/linux/systemd-firstboot.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# systemd-firstboot
+
+> Initialize basic system settings on or before the first boot-up of a system.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-firstboot.html>.
+
+- Set the root directory for the basic system settings initialization:
```
Comment (acuteenvy) [inline pages/linux/systemd-firstboot.md:6]:
```suggestion
- Operate on the specified directory instead of the root directory of the host system:
```

**Review by acuteenvy — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**